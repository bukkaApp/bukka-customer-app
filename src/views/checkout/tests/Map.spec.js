import React from 'react';

import Map from '../common/Map';

describe('Area to explore component', () => {
  const wrapper = mount(<Map />);

  it('renders properly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
