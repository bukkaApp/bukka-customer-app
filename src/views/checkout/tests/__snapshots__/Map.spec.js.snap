// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Area to explore component renders properly 1`] = `ReactWrapper {}`;
